hack/Alfven_speed.py

```diff
@@ -0,0 +1,6 @@
+import numpy as np
+import astropy.units as u
+from astropy.constants.si import mu0
+
+def Alfven_speed(B: u.T, density: (u.m**-3, u.kg / u.m**3)) -> u.m / u.s:
+    return np.abs(B)/np.sqrt(mu0 * density)
```

Comment:
```suggestion
    return np.abs(B)/np.sqrt(mu0 * density)

```

Typical standards require a blank line at the end of a file, that's why you see red minus in a circle above.